data_wrangling/missing.ipynb

```diff
@@ -0,0 +1,997 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        mode = data[idx].mode()[0]
</pre><p>It's difficult to understand what your code is doing at first glance. Please add some comments</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Data-Epic/gbotemi_bolarinwa/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>